mysql-compatibility.md

```diff
@@ -185,3 +185,10 @@ The following column types are supported by MySQL, but **NOT** by TiDB:
 
 + FLOAT4/FLOAT8
 + `SQL_TSI_*` (including SQL_TSI_MONTH, SQL_TSI_WEEK, SQL_TSI_DAY, SQL_TSI_HOUR, SQL_TSI_MINUTE and SQL_TSI_SECOND, excluding SQL_TSI_YEAR)
+
+### Incompatibility caused by deprecated features
+
+TiDB does not implement certain functionality that has been marked as deprecated in MySQL. This includes:
+
+* Specifying precision for floating point types. MySQL 8.0 [deprecates](https://dev.mysql.com/doc/refman/8.0/en/floating-point-types.html) this feature, and it is recommended to use the `DECIMAL` type instead.
+* The `ZEROFILL` attribute. MySQL 8.0 [deprecates](https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html) this feature, and it is recommended to instead pad numeric values in your application.
```

Comment:
```suggestion
* The `ZEROFILL` attribute. MySQL 8.0 [deprecates](https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html) this feature, and it is recommended to use pad numeric values in your application.
```

Reply:
You could say "instead pad numeric values" or "pad numeric values", but not "use pad numberic values". "use pad numeric" would imply this is a feature of an application you could use, but it is not. It is only a broad concept.